packages/riverpod/lib/src/framework/provider_base.dart

```diff
@@ -274,6 +291,12 @@ abstract class ProviderElementBase<State> implements Ref<State>, Node {
   @visibleForTesting
   bool get mounted => _mounted;
 
+  /// Whether this [ProviderElementBase] dependencies have changed or not.
+  ///
+  /// Should not be used, only meant for debug purposes
+  @visibleForTesting
+  bool get dependencyMayHaveChanged => _dependencyMayHaveChanged;
```

Comment:
I don't want this to be visible. If necessary, make the devtool file in the same `part` as this file

Reply:
I'll try making it a part, I tried going with part at first but had some issues. I'll try again and let you know.

Reply:
I've removed the public getter.